CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+#[[
+This source file is part of the Swift System open source project
+
+Copyright (c) 2020 Apple Inc. and the Swift System project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+cmake_minimum_required(VERSION 3.16.0)
+
+project(SwiftMarkdown
+  LANGUAGES C Swift)
+
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules)
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
+
+find_package(ArgumentParser CONFIG REQUIRED)
+include(cmark-gfm)
+
+add_library(CAtomic STATIC
+  Sources/CAtomic/CAtomic.c)
+target_include_directories(CAtomic PUBLIC
+  Sources/CAtomic/include)
+
+add_library(Markdown
+  Sources/Markdown/Base/ChildIndexPath.swift
```

Comment:
Can we use the folder inclusion here? This kind of file enumeration scheme does not look reliable and may be broken in the future.

Reply:
No, globs are not really reliable and make using it far more difficult as they do not pickup changes as they are performed only once.  The CMake recommendation is explicit file lists.

Reply:
Got it. Thanks for the clarification.